pages/golfers/[id].js

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react'
+import { useRouter } from 'next/router'
+import { getToken } from '../../lib/userAuth'
+import Layout from '../../components/Layout'
+import ScoreCard from '../../components/ScoreCard'
+
+export const USER_SCORES_URL = `${process.env.NEXT_PUBLIC_API_URL}/users`
+
+const ProfilePage = () => {
+  const router = useRouter()
+  const { id } = router.query
+
+  const [ scores, setScores ] = useState([])
+  const [ error, setError ] = useState('')
+  const [ userName, setUserName ] = useState('')
```

Comment:
We can eliminate the `useState` and `useEffect` usage by rewriting the hook below.